Marlin/configuration_store.cpp

```diff
@@ -36,10 +36,10 @@
  *
  */
 
-#define EEPROM_VERSION "V23"
+#define EEPROM_VERSION "V24"
```

Comment:
@jbrazio The reason this setting is being added to EEPROM is because it is a configurable setting that can be overridden. Usually, when there's a setting like this where the Configuration value is only supplying a default for something which can be changed later, we also save it in EEPROM and include it in the `M503` output. Of course, in any case I should really name the setting `DEFAULT_HOST_KEEPALIVE_INTERVAL` to make the distinction.

Anyway, if it doesn't seem too vital to save this in EEPROM, then it's no big deal for me to remove that part of this PR.